app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb

```diff
@@ -0,0 +1,279 @@
+module Hyrax
+  module Dashboard
+    module CollectionsControllerBehavior
+      extend ActiveSupport::Concern
+      include Blacklight::AccessControls::Catalog
+      include Blacklight::Base
+
+      included do
+        before_action :filter_docs_with_read_access!, except: :show
+        before_action :remove_select_something_first_flash, except: :show
+
+        include Hyrax::Collections::AcceptsBatches
+
+        # include the render_check_all view helper method
+        helper Hyrax::BatchEditsHelper
+        # include the display_trophy_link view helper method
+        helper Hyrax::TrophyHelper
+
+        # This is needed as of BL 3.7
+        copy_blacklight_config_from(::CatalogController)
+
+        # Catch permission errors
+        rescue_from Hydra::AccessDenied, CanCan::AccessDenied, with: :deny_collection_access
+
+        # actions: index, create, new, edit, show, update, destroy, permissions, citation
+        before_action :authenticate_user!, except: [:index]
+
+        class_attribute :presenter_class,
+                        :form_class,
+                        :single_item_search_builder_class,
+                        :member_search_builder_class
+
+        alias_method :collection_search_builder_class, :single_item_search_builder_class
```

Comment:
Do you need an alias on the class methods as well? What about the setter methods?